src/components/rpc_base/test/rpc_base_json_test.cc

```diff
@@ -124,7 +124,7 @@ TEST(ValidatedTypesJson, IntegerAbsentValueTest) {
 }
 
 TEST(ValidatedTypesJson, IntegerFromOverflowingJsonTest) {
-  Value int_val(0xFFFFFFFFFFll);
+  Value int_val(0xFFFFFFFFFFl);
```

Comment:
Can you please explain why you made this change?

Reply:
@iCollin call of overloaded `Value(long long int)` is ambiguous in new library, because of that I changed the type to a more suitable one.

Reply:
How will this work in a 32 bit build where long is 4 bytes?

Reply:
@iCollin SDL is tested on a 64-bit system and as far as I know car manufacturers no longer use a 32-bit system